src/VscProcess.cpp

```diff
@@ -176,28 +163,35 @@ void VscProcess::checkCharacterLimit(const hri_safe_remote_control_system::SrcDi
 
 void VscProcess::receivedDisplayOnCommand(const hri_safe_remote_control_system::SrcDisplay& msg)
 {
-  // Turn on custom display mode
-  vsc_send_user_feedback(vscInterface, VSC_USER_DISPLAY_MODE, DISPLAY_MODE_CUSTOM_TEXT);
+  if (vscInterface != NULL)
```

Comment:
just return if vscInterface == NULL rather than nesting the regular case. A few other places as well.

Reply:
Switched NULL check logic where appropriate.